darts/explainability/shap_explainer.py

```diff
@@ -395,6 +395,11 @@ def summary_plot(
             for the sake of performance.
         plot_type
             Optionally, specify which of the shap library plot type to use. Can be one of ``'dot', 'bar', 'violin'``.
+
+        Returns
```

Comment:
Shall we just return the `shaps_` dict?

Might be a bit cumbersome to interpret which element belongs to which horizon/components

Reply:
Good idea, changed it, updated the docstring and added a little test.